Backpack/Calendar/Classes/BPKCalendarCell.m

```diff
@@ -212,4 +212,8 @@ - (void)setSelectionType:(SelectionType)selectionType {
     }
 }
 
+-(void)configureWithData:(id)data {
+    return;
```

Comment:
Since this is an abstract base class we should have an assert here or throw an exception